vars/buildDaosJob.groovy

```diff
@@ -0,0 +1,32 @@
+// vars/buildDaosJob.groovy
+
+  /**
+   * buildDaosJob step method.
+   *
+   * This is intended only for use in a Jenkinsfile in a Matrix stage.
+   * Builds a branch of DAOS based on env.TEST_BRANCH.
+   *
+   * The paramater names must match parameters that are the Jenkinsfile for
+   * DAOS master.
+   *
+   * @param config Map of parameters passed
+   *
+   */
+
+Map call(Map config = [:]) {
+    build job: 'daos-stack/daos/' + setupDownstreamTesting.test_branch(env.TEST_BRANCH),
+                parameters: [string(name: 'TestTag',
+                             value: 'load_mpi test_core_files'),
+                             // Maybe should only do this if Test-tag: provisioning?
+                             string(name: 'CI_RPM_TEST_VERSION',
+                                          value: daosLatestVersion(env.TEST_BRANCH)),
+                             string(name: 'BuildPriority', value: '2'),
```

Comment:
A few things seemed to have gotten lost in the functionalization:

```suggestion
                parameters: [string(name: 'TestTag',		
                                    value: cachedCommitPragma(		
                                      'Test-tag',		
                                      'load_mpi test_core_files ' +		
                                      'test_pool_info_query')),		
                             string(name: 'CI_RPM_TEST_VERSION',		
                                    value: cachedCommitPragma('Test-skip-build', 'false') == 'true' ?		
                                             daosLatestVersion(env.TEST_BRANCH) : ''),		
                             string(name: 'BuildPriority', value: priority),		
```

Reply:
Changed in current push